packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -48,35 +69,76 @@
             }
             &--textarea {
                 height: 9.6rem;
+
                 margin-bottom: 0;
+
+                @include mobile {
+                    width: 100%;
+                }
+
+                .dc-input__hint {
+                    top: 9.7rem;
+                }
+                .dc-input__counter {
+                    top: 9.7rem;
+                    right: 0rem;
+                }
+            }
+
+            @include mobile {
+                margin-bottom: unset;
             }
+
             &--trailing {
                 font-size: var(--text-size-xxs);
                 color: var(--text-less-prominent);
+
+                @include mobile {
+                    color: var(--text-prominent);
+                    font-size: var(--text-size-s);
+                }
```

Comment:
by using the Text component you can remove this calss
```suggestion
      
```

Reply:
Same as above.